src/modules/facility/facility.controller.ts

```diff
@@ -0,0 +1,66 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiInternalServerErrorResponse, ApiNotFoundResponse, ApiOkResponse, ApiOperation, ApiParam } from '@nestjs/swagger';
+
+import { GetFacilityByIdentifierResponseDto } from './dto/get-facility-by-identifier-response.dto';
+import { FacilityService } from './facility.service';
+
+@Controller('facilities')
+export class FacilityController {
+  constructor(private readonly facilityService: FacilityService) {}
+
+  @Get(':facilityIdentifier')
+  @ApiOperation({ summary: 'Get the facility matching the specified facility identifier.' })
+  @ApiParam({
+    name: 'facilityIdentifier',
+    required: true,
+    type: 'string',
+    description: 'The identifier of the facility in ACBS.',
+    example: '0000000001',
```

Comment:
Can we move this to the EXAMPLES constant in `src/constants/examples.constant.ts` please?